README.md

```diff
@@ -1,70 +1,79 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Countries Information
 
-## Available Scripts
+> Description the project.
+Countries Information is a simple react capstone project that displays information about each country around the world. Built with react 
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- JavaScript
+- CSS
+- React
+- Redux
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Live Demo (if available)
 
-### `npm test`
+[link](mellifluous-begonia-3df97c.netlify.app)
```

Comment:
- [ ] Please know that when you click the link it doesn't direct you to the netlify app, that is because you didn't include the full URL which should be https://mellifluous-begonia-3df97c.netlify.app/ kindly change this so that the link in your README is working correctly.